resqpy/rq_import/GridFromCp.py

```diff
@@ -0,0 +1,555 @@
+"""GridFromCp.py: Class to generate a RESQML grid object from an input corner point array, and other optional inputs."""
+
+version = '15th November 2021'
+
+# Nexus is a registered trademark of the Halliburton Company
+
+import logging
+
+log = logging.getLogger(__name__)
+log.debug('GridFromCp.py version ' + version)
+
+import numpy as np
+import numpy.ma as ma
+
+import resqpy.crs as rqc
+import resqpy.grid as grr
+import resqpy.olio.vector_utilities as vec
+
+
+class GridFromCp:
+    """Class to build a resqpy grid from a Nexus CORP array"""
```

Comment:
I suggest making this a private class (_GridFromCp) and revealing it publicly be an @classmethod in the Grid class, which would end up returning the _GridFromCp.grid attribute.  Let's discuss.

Reply:
I'm going to leave this for now to avoid conflicts with @cflynn3 's grid.py refactoring, but will add as an issue for later consideration.